studygroups/views/facilitate.py

```diff
@@ -108,12 +108,9 @@ def view_study_group(request, study_group_id):
 
 
 class FacilitatorRedirectMixin(object):
-    """ Redirect facilitators back to their dashboard & organizers to the study group page """
+    """ Redirect to the study group page """
 
     def get_success_url(self, *args, **kwargs):
-        study_group = get_object_or_404(StudyGroup, pk=self.kwargs.get('study_group_id'))
-        if study_group.facilitator == self.request.user:
-            return reverse_lazy('studygroups_facilitator')
         return reverse_lazy('studygroups_view_study_group', args=(self.kwargs.get('study_group_id'),))
```

Comment:
@dirkcuys are you ok with this? Facilitators will get redirected back to the studygroup page whenever they edit anything on it, instead of getting redirected to the dashboard.

Reply:
Yes, I think that makes sense. Before this logic was needed because organizers could make edits and it didn't make sense to send them back to their dashboard where they won't see the learning circle they've edited. Now this happens for everyone, so they should go to the studygroup page.